src/components/core/header/app-header.tsx

```diff
@@ -51,7 +52,9 @@ export const AppHeader: FunctionComponent = () : ReactElement => {
             });
         getAuthenticators("idp")
             .then((response) => {
-                setAuthFactors((elements: any)=>[...elements, ...response.data.identityProviders]);
+                if (response.data.identityProviders) {
```

Comment:
What should be the value of `authFactors` if `response.data.identityProviders` is not defined?

Reply:
Fixed with [a68afe2](https://github.com/wso2-incubator/identity-authentication-flow-composer/pull/20/commits/a68afe278686c3df15a57d4748efed6b1fe75f06)